### DISCUSSION TITLE
Skipped directories being deleted from OneDrive

### DISCUSSION THREAD

**Discussion by Neriderc** [bpozdena/OneDriveGUI#196] [Q&A] [answered]
After updating to the latest major version, the excluded directories I have defined in the profile->excluded files->skip directories section are now being removed from OneDrive instead of being ignored for the sync.

Has something changed in this? I have them set up like this:

![image](https://github.com/user-attachments/assets/9034f069-c532-40f1-accd-c7bd1129f019)

Have I got this right?

When I restore from the OneDrive recycle bin, they get deleted again on the next pass.

**Comment by abraunegg** [comment, marked as answer]
@Neriderc 
Sounds like an *application* issue not a *GUI* issue ........

Confirm what the 'onedrive' application is *actually* doing in the background. Once you have determined this, raise a query and/or bug here: https://github.com/abraunegg/onedrive

**Comment by abraunegg** [comment]
@Neriderc 
Confirmed a bug as per https://github.com/abraunegg/onedrive/issues/2910

The fix is also available if you are able to assist in testing

**Reply by Neriderc** [reply]
Thanks, I saw this but haven't been able to reliably reproduce this ever since I removed my profile and started again (after posting this originally). I have struggled to get a full sync, with crashes and it keeps warning about an application change where it needs to re-sync (I didn't change anything, but it's probably a result of the crash). It then takes 2 days or more to get back to where it was because I have too many files.

I have seen this issue once after re-creating my profile, but I'm still slowly trying to work out what the actual trigger is.
